markdown/articles/2020/03/22/DDFAQs.md

```diff
@@ -0,0 +1,600 @@
+the variants out so that people could read a summary of each and select
+the one/(s) they liked the sound of to have access to those select
+parts of the server. Despite these summaries being present we still get
+some confusion as to what the differences actually entail both in terms
+of deck building decisions and in terms of personal preferences or
+gameplay styles.
+
+I will present the three main archetypes below with the current Discord 
+summary and a bit more comparison on each. 
+
+
+
+### 1.1 DDFT (DoomsDay Fetchland Tendrils)
+
+Discord Summary: *Lists that utilise traditional elements such as
+Burning WIsh, some sort of "draw 3/4" engine and focus on a Storm
+style approach to winning (however do not always require storm to win).
+
+[As mentioned last time](/article/2020/03/12/pretty-mean-deck.md), DDFT
+is the traditional name of historical iterations of the deck. We won't
+do a full history lesson here however it basically refers to the fact
+that this is a storm deck that can play a much longer game if required.
+
+The deck normally incorporates some sort of raw card advntage engine
+such as {{Ideas Unbound}}, {{Act on Impulse}}, {{Three Wishes}} or 
+{{Cruel Bargain}} to enable drawing into the Doomsday pile. {{Burning Wish}}
+has been used often to enable additional pseudo copies of Doomsday and 
+newer cards like {{Echo of Eons}} ave been incorporated in lists too.
+The aim of the deck is to kill via a Doomsday pile with lethal
+{{Tendrils of Agony}} or {{Thassa's Oracle}} or a natural Tendrils
+storm kill without Doomsday. See an example pile below:
+
+<pile>{{!AOI}} {{!LED}} {{!ToA}} {{!PN}} {{!CB}}</pile>
+
+Due to the cards used, the deck tends to be Grixis colours sometimes
+splashing Green for sideboard options.
+
+### 1.2 DDEFT (Doomsday Experimental Frenzy Tendrils)
+
+Discord Summary: *Lists that utilise Experimental Frenzy as the primary
+engine to win. 
+
+The summary defines this sub-archetype pretty well. {{Experimental Frenzy}}
+is a very powerful card advantage engine that can, under the right
+circumstances, allow you to transform your entire deck into a Doomsday pile.
+If you cast a Doomsday with Frenzy active, you have a deterministic kill that
+can get through infinite life should you wish. The deck can pull off the same
+win methods as DDFT but also allows winning via infinite copies of 
+{{Collective Brutality}} amongst other things (see example below).
+
+<pile>{{!LED}} {{!LED}} {{!LED}} {{!CoBru}} {{!DD}}</pile>
+
+Instead of using {{Burning Wish}} for additional copies of Doomsday, the deck
+uses cards like {{Lim-Dul's Vault}} to act as tutor pieces prior to "going off"
+and facilitate the kill from an active Frenzy (as they let you stack the top
+of the deck you are playing from).
+
+Due to the cards used, this deck also tends to be Grixis colours sometimes
+splashing Green for sideboard options.
+
+### 1.3 Meandeck Doomsday (We couldn't decide on a good acronym for this one)
```

Comment:
I'd suggest pointing out that this is the version that has been seeing recent success, since most of the new people probably care the most about this version.

Reply:
Added comment.